### PR TITLE
Email verification

### PR DESCRIPTION
**Pull request by Singletoned**

### CODE REVIEW

File: organogram/organogram/verification.py [hunk 1]
```diff
@@ -0,0 +1,44 @@
...
+
+TOKEN_GENERATOR = PasswordResetTokenGenerator()
+EMAIL_SUBJECT = "Verify your email"
+EMAIL_TEMPLATE = """
+Your email {user.email} address was entered in the Cabinet Office Organogram.
+
+If it was you that did this, please confirm your email at the following url:
+
+{url}
+
```
Comment (nmenezes0) [inline organogram/organogram/verification.py:18]: Do you need to say what to do if they didn't do it (eg "ignore if you didn't")?

File: organogram/organogram/verification.py [hunk 2]
```diff
@@ -0,0 +1,44 @@
...
+Thank you
+"""
+
+
+def send_verification_email(user):
+    token = TOKEN_GENERATOR.make_token(user)
+    host_url = settings.HOST_URL.strip("/")
+    url = "/".join(("http:/", host_url, "user", str(user.id), "verify", token))
+    body = EMAIL_TEMPLATE.format(user=user, url=url)
+    send_mail(
```
Comment (nmenezes0) [inline organogram/organogram/verification.py:28]: I tested the link that was printed out in the console (eg `http://localhost:8001/user/f036ce71-a7e6-432c-9c4d-955f0c4b0fe9/verify/b9jeus-cef58722a1bf6ede5d403d4e15ce296f`) but got this error when I tried to access it:
```organogram_1        | Internal Server Error: /your-details
organogram_1        | Traceback (most recent call last):
organogram_1        |   File "/usr/local/lib/python3.8/site-packages/django/core/handlers/exception.py", line 47, in inner
organogram_1        |     response = get_response(request)
organogram_1        |   File "/usr/local/lib/python3.8/site-packages/django/core/handlers/base.py", line 181, in _get_response
organogram_1        |     response = wrapped_callback(request, *callback_args, **callback_kwargs)
organogram_1        |   File "/usr/local/lib/python3.8/contextlib.py", line 75, in inner
organogram_1        |     return func(*args, **kwds)
organogram_1        |   File "/app/organogram/organogram/page_views.py", line 374, in page_view
organogram_1        |     return view_map[page_name](request, url_data)
organogram_1        |   File "/app/organogram/organogram/page_views.py", line 130, in your_details_view
organogram_1        |     data = model_to_dict(request.user)
organogram_1        |   File "/usr/local/lib/python3.8/site-packages/django/forms/models.py", line 85, in model_to_dict
organogram_1        |     opts = instance._meta
organogram_1        |   File "/usr/local/lib/python3.8/site-packages/django/utils/functional.py", line 247, in inner
organogram_1        |     return func(self._wrapped, *args)
organogram_1        | AttributeError: 'AnonymousUser' object has no attribute '_meta'```

**Review by nmenezes0 — COMMENTED** [review]
See comments - basically I can't access link to verify email (should I be able to?)

File: organogram/organogram/verification.py (hunk 2 above)
Comment (Singletoned) [reply organogram/organogram/verification.py:28]: It worked, but the page that it redirected to had an error.  Check the user and you will see that it now has `verified=True` on it

File: organogram/organogram/verification.py (hunk 1 above)
Comment (Singletoned) [reply organogram/organogram/verification.py:18]: No idea about that.  Need to get someone who knows what they are talking about to write it.  Feel free to improve it yourself

Comment (nmenezes0) [reply organogram/organogram/verification.py:19]: I'll leave as is for now - if someone complains we can change (I don't actually know what to do if they get it in error - they would probs email us).

File: organogram/organogram/verification.py (hunk 2 above)
Comment (nmenezes0) [reply organogram/organogram/verification.py:29]: Yup - looks good (didn't think that error was caused by your stuff but still thought I needed to point it out!).

Also, I added that field to the admin to make my life easier.

**Review by nmenezes0 — APPROVED** [review]
Looks good.

I also added to the admin - so maybe check my change before merging.

**Pull request merged**